SConstruct

```diff
@@ -1984,11 +2022,11 @@ def postInstallMessage(target, source, env):
 
         File locations:
 
-          applications                {ct_bindir!s}
-          library files               {ct_libdir!s}
-          C++ headers                 {ct_incroot!s}
-          samples                     {ct_sampledir!s}
-          data files                  {ct_datadir!s}""".format(**env_dict))
+          applications                {inst_bindir!s}
+          library files               {inst_libdir!s}
+          C++ headers                 {inst_incroot!s}
+          samples                     {inst_sampledir!s}
+          data files                  {inst_datadir!s}""".format(**env_dict))
```

Comment:
I suspect that these were the `ct`-prefixed variables because the `inst` variables include the stage, which hasn't been printed here in the past. I'm fine either way 🤷‍♂️

Reply:
Correct. From what I understand, this should be `stage`.

Reply:
Prior to this PR, the paths were all incorrect if `stage_dir` is specified. On `*nix`, the installation appears to be successful though. As this is a pre-existing bug that also installs Python into an incorrect folder on Windows, I am not fixing this here.